calendarsubscription/calendarsubscription.go

```diff
@@ -0,0 +1,31 @@
+package calendarsubscription
+
+import (
+	"github.com/target/goalert/validation/validate"
+	"time"
+)
+
+type CalendarSubscription struct {
+	ID         string    `json:"id"`
```

Comment:
I think we can remove the `json:""` definitions, we only had those in older stuff because we needed it for the legacy gql API. New stuff should work by default.